docs/source/configuration/traffic-shaping.mdx

```diff
@@ -2,19 +2,8 @@
 title: Traffic shaping in the Apollo Router
 ---
 
-The Apollo Router supports the following types of traffic shaping between itself and your subgraphs:
-
-- **Sub-query deduplication** - Whenever the router is sending multiple identical in-flight query operations to a subgraph, it can consolidate them into a single request.
-  - Mutation operations are never deduplicated.
-  - Only in-flight requests are deduplicated.
-- **Variable deduplication** - If a request to a subgraph includes multiple GraphQL variables with the same value, the router can replace those with a single variable.
-- **Compression** - The router can compress request bodies to subgraphs (along with response bodies to clients) with a supported algorithm
-  - The router currently supports `gzip`, `br`, and `deflate`.
-- **Global rate limiting** - If you want to rate limit requests to subgraphs or to the router itself.
-- **Timeout**: - Set a timeout to subgraphs and router requests.
-- **Request Retry**: - retry subgraph requests if they fail due to network errors. This implements Finagle's retry buckets mechanism. **Experimental feature**: retry configuration might change in the future.
-
-Each of these optimizations can reduce network bandwidth and CPU usage for your subgraphs.
+The Apollo Router provides various features to improve the performance and reliability of
+the traffic with clients and subgraphs.
```

Comment:
```suggestion
the traffic between clients and subgraphs.
```

Reply:
This is about the traffic between client and router, and between router and subgraph

Reply:
Yep. I was trying to be more clear, but perhaps we just need to be more explicit, e.g.:
```
the traffic between the client and router and between the router and subgraphs.
```
?

Reply:
fixed in 4e3f567b